### DISCUSSION TITLE
asn to CIDRs

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/mapcidr#105] [Q&A] [answered]
Hi can this tool expand an asn to CIDRs ? i don't see such option

if so, what's the command

thanks!

**Comment by ehsandeep** [comment, marked as answer]
It will be supported soon @molitona :)

**Comment by molitona** [comment]
ok, thanks!!

**Comment by molitona** [comment]
do you know any other tool/script that does it as a temporarily solution ?
